### DISCUSSION TITLE
ERR: path must end with a slash (test)

### DISCUSSION THREAD

**Discussion by Torstein-Eide** [bluenviron/mediamtx#405] [Questions]
there is a  two contradictory errors:
````python
2021/05/20 13:33:36 [I] [1/0/0] [client 10.0.0.1:52672] ERR: path must end with a slash (test)
2021/05/20 13:33:42 [I] [1/0/0] [client 10.0.0.1:52676] ERR: invalid path name: can't end with a slash (test/)
````
config used:
````yml
  test:
    source: rtsp://10.0.0.3/axis-media/media.amp
````

````python
2021/05/20 13:33:36 [I] [1/0/0] [client 10.0.0.1:52672] ERR: path must end with a slash (test)
2021/05/20 13:33:36 [I] [0/0/0] [client 10.0.0.1:52672] disconnected
2021/05/20 13:33:36 [I] [1/0/0] [client 10.0.0.1:52674] connected (RTSP/TCP)
2021/05/20 13:33:36 [I] [0/0/0] [client 10.0.0.1:52674] disconnected
2021/05/20 13:33:36 [I] [0/0/0] [path test] [rtsp source] ERR: dial tcp 10.0.0.3:554: i/o timeout
2021/05/20 13:33:41 [I] [0/0/0] [path test] [rtsp source] connecting
2021/05/20 13:33:42 [I] [1/0/0] [client 10.0.0.1:52676] connected (RTSP/TCP)
2021/05/20 13:33:42 [I] [1/0/0] [client 10.0.0.1:52676] ERR: invalid path name: can't end with a slash (test/)
````

**Comment by dorinclisu** [comment]
First of all, you should not expect a solution (or even a response) after failing to adhere to the issue template. It shows a disrespect for the contributors' time.

You couldn't even bother to mention the version of the server.

**Comment by Torstein-Eide** [comment]
Hi @dorinclisu   
You are correct, for some reason the issue template was not used.  I will update with all the necessary information.

**Comment by aler9** [comment]
Hello, what happens is the following:

1. the server tries to establish a connection to the source (the camera), but it can't since the camera can't be reached (`dial tcp 10.0.0.3:554: i/o timeout`)
2. a client (VLC?) tries to connect to the server, but the server hasn't established a connection to the source yet, and  it rejects the client; VLC tries a fallback, unsupported protocol and this generates the error `path must end with a slash (test)`
3. another client tries to connect to the server with a final slash, but this is not supported and generated the error `invalid path name: can't end with a slash (test/)`

Make sure that the server can reach the camera and wait for the log message

```
[rtsp source] ready
```

[a comment by an automated account is not shown]
